pagopa/pn-frontend
# Fix/pn 12909 - Prevent user from enabling/disabling Sercq while validating PEC address

## Short description
The code contained in this PR aims to prevent the user from modifying sercq value for default and custom contacts while there's a PEC address which needs to be validated. Changes involved both PF and PG webapps.

## List of changes proposed in this pull request
- code to prevent the user to enable/disable sercq while validating a pec address for the same entity
- changes to contacts.utility.ts to fix a bug which was causing adding a new contact instead of replacing the old one on specific cases
- banner for validating address
- unit tests

## How to test
Run either PF or PG app and verify the that the UI behaviour follows what described on the corresponding Jira task for both default and special legal contacts
